php/php.editor/src/org/netbeans/modules/php/editor/api/PhpModifiers.java

```diff
@@ -132,8 +132,8 @@ private PhpModifiers(int... bitmask) {
         for (int mode : bitmask) {
             this.mod |= mode;
         }
-        if (!Modifier.isPrivate(mod) && !Modifier.isProtected(mod) && !Modifier.isImplicitPublic(mod)) {
-            mod |= Modifier.PUBLIC;
+        if (!Modifier.isPrivate(mod) && !Modifier.isProtected(mod) && !Modifier.isPublic(mod)) {
+            mod |= Modifier.IMPLICIT_PUBLIC;
         }
```

Comment:
`IMPLICIT_PUBLIC` is better if `mod` is `0` because the public modifier doesn't exist.